contrib/jenkins_tests/test.sh

```diff
@@ -80,6 +79,20 @@ for test_link in $test_ip_list; do
 	done
 done
 
+
+test_ip_list="/tmp/sockperf-test-$$"
+test_list="uds"
+
+for test_link in $test_ip_list; do
+	for test in $test_list; do
+		test_in="uds"
+		test_name="uds_test"
+		$timeout_exe ${WORKSPACE}/tests/verifier/verifier.pl -a ${test_app} \
+				-t ${test_in} -s ${test_link} -l ${test_dir}/${test_name}.log \
+				--progress=0
+	done
+done
```

Comment:
Could you please follow recommendation https://github.com/Mellanox/sockperf/pull/200#discussion_r956244189
It allows to do many things that are required.